src/should-forward-property.js

```diff
@@ -600,17 +26,18 @@ const isCustomAttribute = RegExp.prototype.test.bind(
 )
 
 const hasItem = (list, name) => list.indexOf(name) !== -1
-const isHtmlProp = name => hasItem(htmlProps, name)
+const isHtmlProp = (name, tagName) => {
+  const elementAttributes = reactHTMLAttributes[tagName]
+
+  return hasItem([...globalReactHtmlProps, ...elementAttributes], name)
```

Comment:
And I think that gzip size isn't even minified. That's awesome. Looks perfect. I say it's worth the bytes :+1:

Feel free to implement it as part of this PR or a new one 👍